decision-making/voting-on-proposals.md

```diff
@@ -0,0 +1,30 @@
+# Decision-making
+
+### Proposal submission
+Proposal is submitted by opening pull request to this repository.
+Initiator must present the idea on the architecture meeting.
+The announcement should include problem description the proposal is trying to resolve as well as solution(s).
```

Comment:
So does the proposal actually need to contain code? Or descriptions of the solution? A PR template might be helpful to not forget things

Reply:
PR is a document or a set of documents. There is a template with suggested structure at https://github.com/magento/architecture/blob/master/design-documents/README.md